system/jlib/jsocket.cpp

```diff
@@ -5010,37 +4988,34 @@ class CSocketEpollThread: public CSocketBaseThread
                                     continue;
                                 }
 # endif
-                                if (!epsi->del)
+                                if (!epsi->sock || !epsi->nfy)
```

Comment:
Is this ever possible? Should be validated on add and be impossible I'd have thought, so check here shouldn't be necessary.

Reply:
ok, removed